week1/project/ecommerce/src/App.jsx

```diff
@@ -0,0 +1,32 @@
+import { useState } from "react";
+import reactLogo from "./assets/react.svg";
+import viteLogo from "/vite.svg";
+import "./App.css";
+import productsData from "./fake-data/all-products";
+import categories from "./fake-data/all-categories";
+import CategoryList from "./CategoryList";
+import ProductList from "./ProductList";
+
+function App() {
+  const [activeCategory, setActiveCategory] = useState("All");
+
+  const filteredProducts =
+    activeCategory === "All"
+      ? productsData
+      : productsData.filter((product) => {
+          return product.category === activeCategory.replace("FAKE: ", "");
```

Comment:
```suggestion
          return product.category === activeCategory;
```

Suggestion: 
`.replace("FAKE: ", "")` ideally be handled within `ProductList`.
Due to the Single Responsibility principle, The App pages shouldn't need to know the 'FAKE:' detail.